Xamarin.Forms.Core.UnitTests/ShellTests.cs

```diff
@@ -764,6 +764,99 @@ public async Task FlyoutNavigateToImplicitContentPage()
 
 		}
 
+		[Test]
+		public void FlyoutGroupsNumbersForDifferentFlyoutDisplayOptions()
+		{
+			var shell = new Shell();
+			var shellItem = new ShellItem() { FlyoutDisplayOptions = FlyoutDisplayOptions.AsMultipleItems, };
+			var shellItem2 = new ShellItem();
+			var shellSection = new ShellSection() { Title = "can navigate to" };
+			shellSection.Items.Add(new ContentPage());
+
+			var shellSection2 = new ShellSection() { Title = "can navigate to" };
+			shellSection2.Items.Add(new ContentPage());
+
+			var implicitSection = CreateShellSection(new ContentPage(), asImplicit: true);
+
+			shellItem.Items.Add(shellSection);
```

Comment:
Can you change this to not reuse the same shellSection class on different Items? On the other tests as well?

Also there are a number of base methods you can use to reduce some of the code

```C#
var shellSection = new ShellSection() { Title = "can navigate to" };
			shellSection.Items.Add(new ContentPage());
```

Can just be
```C#
var shellSection = CreateShellSection();
```

Reply:
done